provider/cmd/pulumi-resource-aws/schema.json

```diff
@@ -9696,11 +9696,8 @@
         },
         "aws:codebuild/ProjectSecondarySource:ProjectSecondarySource": {
             "properties": {
-                "auths": {
-                    "type": "array",
-                    "items": {
-                        "$ref": "#/types/aws:codebuild/ProjectSecondarySourceAuth:ProjectSecondarySourceAuth"
-                    },
+                "auth": {
```

Comment:
@mikhailshilkov the issue here is that the upstream provider implemented MaxItems: 1

This is technically a breaking change as it changes the parameter name for our users

https://github.com/terraform-providers/terraform-provider-aws/commit/dd9af8653cfc0bc6ecfeeb536946b891158ce69d

Reply:
Fixed!